sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveContext.scala

```diff
@@ -549,9 +549,11 @@ class HiveContext(sc: SparkContext) extends SQLContext(sc) with Logging {
   }
 
   protected[hive] def runSqlHive(sql: String): Seq[String] = {
-    if (sql.toLowerCase.contains("create temporary function")) {
+    val normalized = sql.trim.toLowerCase
+    if (normalized.contains("create temporary function") ||
+      normalized.contains("create temporary macro")) {
```

Comment:
We don't actually support macros right now, right? I'd prefer to revert this change and handle it separately as part of your macro PR.